obkvrpc/connection_pool.go

```diff
@@ -71,10 +74,12 @@ func NewPoolOption(ip string, port int, connPoolMaxConnSize int, connectTimeout
 
 func NewConnectionPool(option *PoolOption) (*ConnectionPool, error) {
 	pool := &ConnectionPool{
-		option:      option,
-		connections: make([]*Connection, 0, option.connPoolMaxConnSize),
-		rwMutexes:   make([]sync.RWMutex, 0, option.connPoolMaxConnSize),
+		option:            option,
+		connections:       make([]*Connection, 0, option.connPoolMaxConnSize),
+		rwMutexes:         make([]sync.RWMutex, 0, option.connPoolMaxConnSize),
+		disConnectedCount: atomic.Int32{},
```

Comment:
I guess connectedCount may be a better choice since we could easily know the number of active connections.